dali/test/python/decoder/test_video.py

```diff
@@ -64,13 +64,14 @@ def video_loader(batch_size, epochs):
         yield batch
 
 
-def video_decoder_iter(batch_size, epochs=1, device="cpu"):
+def video_decoder_iter(batch_size, epochs=1, device="cpu", module=fn.experimental):
```

Comment:
What is the module for, if there's only one value ever used? Do you predict that we'll have two different modules at some point?

Reply:
I was hoping that when we have the plugin with the alternative decoder, we can easily reuse this test to test that one, as they have the same API